core/controllers/admin.py

```diff
@@ -848,3 +848,16 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class GetInteractionsByExplorationId(base.BaseHandler):
+    """This function returns a list of interaction id's for a exploration id."""
```

Comment:
Nit on documentation: this is a handler, not a function. See other handlers above for how they specify their KDoc.

Reply:
Will change

Reply:
@aggarwalpulkit596, is this comment resolved? (If yes then make sure to add a reply which states that this comment is addressed (Like "Done!")

Reply:
oh yes i forgot this one